RCPTT_Tests/BlocksTests/BlocksAreFormattedCorrectly.test

```diff
@@ -0,0 +1,54 @@
+--- RCPTT testcase ---
+Format-Version: 1.0
+Element-Name: BlocksAreFormattedCorrectly
+Element-Type: testcase
+Element-Version: 3.0
+External-Reference: 
+Id: _Vcj1YFEBEeeEZqjlkcOTbQ
+Runtime-Version: 2.1.0.201606221726
+Save-Time: 6/14/17 3:54 PM
+Testcase-Type: ecl
+
+------=_.content-0a7243a0-75d3-3d5f-9791-539de0e5b7ac
+Content-Type: text/ecl
+Entry-Name: .content
+
+proc configure_simple_ioc [val name] {
+	let [val window [edit_current_config]] {
+ 		add_ioc -ioc_name "SIMPLE" -window $window
+ 		with [get_ioc_dialog] {
+	 		get-button "Auto-Start" | check
+	     	get-combo -after [get-label "Sim. Level"] | select RECSIM
```

Comment:
I don't think this is needed. `SIMPLE` IOC doesn't connect to hardware, it's purely software, so works just fine in normal mode